tests/backup/backup_delete_test.go

```diff
@@ -503,7 +502,7 @@ var _ = Describe("{DeleteBucketVerifyCloudBackupMissing}", func() {
 			log.FailOnError(err, "Fetching px-central-admin ctx")
 			for _, provider := range providers {
 				cloudAccountName = fmt.Sprintf("%s-%s-%v", "cloudcred", provider, time.Now().Unix())
-				bkpLocationName = fmt.Sprintf("%s-%s-%v-bl", provider, getGlobalBucketName(provider), time.Now().Unix())
+				bkpLocationName = fmt.Sprintf("%s-%s-%v-bl", provider, "local-location", time.Now().Unix())
```

Comment:
Use a unique name to be safe. Timestamp or random string.

Reply:
Didn't get your point, we are already adding timestamp as suffix for bkpLocationName .

Reply:
Sorry, my bad